src/components/shared/ReusableLeagueTable.tsx

```
@@ -0,0 +1,772 @@
+
+      // Za ostale lige (H2H, H2H2) - prvo pokazuj trofeje/medalje za top 3
+      switch (position) {
+        case 1:
+          return <Trophy className="w-5 h-5" style={{ color: "#FFD700" }} />;
+        case 2:
+          return <Medal className="w-5 h-5" style={{ color: "#C0C0C0" }} />;
+        case 3:
+          return <Medal className="w-5 h-5" style={{ color: "#CD7F32" }} />;
+        default:
+          // Za pozicije 4+ proverava da li ima novčanu nagradu
+          const prize = prizes.find((p) => p.position === position);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)